docusaurus/docs/configuration/controller.md

```diff
@@ -0,0 +1,526 @@
+on controller crash or kill. Decreasing it will increase the frequency of database writes. The default should be
+sufficient.
+
+The `address` setting is unique as it must match the `address` in a `bindPoint` for the `edge-client` API. This is to
+ensure that responses and data persisted outside the system can reach the controller. An example of this is
+enrollment JWTs that contain the URL that is used to complete enrollment via the Edge Client API.
+
+#### `enrollment`
+
+The `enrollment` section under `edge` defines values that pertain specifically to identity and router enrollment. This
+includes the certificate and private key used to sign certificates as well as enrolment JWT lifetimes.
+
+The enrolment section has these subsection:
+
+- `signingCert` - (required)  defines the certificate and key used to sign identity and router certificates
+- `edgeIdentity` - (optional) controls identity enrollment options
+- `edgeRouter` - (optional) controls router enrollment options
+
+#### `signingCert`
+
+An object defining the `cert` and `key` used to issue certificates to identities and routers.
+
+- `cert` - (required) the x509 PEM formatted certificate used to sign certificate, must be a root or intermediate CA
+- `key` - (required) the x509 PEM formatted private key used to sign certificates, must be the key for the certificate
+  defined
+  in `cert`
+
+As this signing certificate will be the signer for all issued edge router certificates, it is important that all
+enrolling identities and routers trust the PKI that issued the signing certificate. To have that happen the trust anchor
+of the PKI the `signingCert` is a member of should be in the `ca` bundle defined in the controller's `identity` section.
+The `ca` value is used as part of the trust bundle delivered to identities and routers during enrollment.
+
+#### `edgeIdentity`
+
+The `edgeIdentity` section controls the lifetime of identity enrollment JWTs. It has only one value:
+
+- `duration` - (optional) the lifetime of identity enrollment JWTs
+
+#### `edgeRouter`
+
+The `edgeRouter` section controls the lifetime of router enrollment JWTs. It has only one value:
+
+- `duration` - (optional) the lifetime of router enrollment JWTs
+
+### `events`
+
+The `events` section allows for the definition of multiple event loggers with their own handler and event subscriptions.
+Handlers define the type, format, and destination for events. Subscriptions handle which events are routed to the
+handler. This allows different events to be output in different manners or to different locations.
+
+The `events` section is an array of named objects. The name (`jsonLogger` in the example below) is used for
+configuration error output only. Each logger has a `subscriptions` and `handler` section. The `subscriptions` section is
+an array of objects with fields associated with the event type. The list of valid event types and their options is as
+follows:
+
```

Comment:
Maybe quick note indicating that all options are optional? Not sure if that's necessary or not